### PR TITLE
feature/103

### LINKED ISSUE #103: New Airport: Tokyo Narita International Airport
See openscope/openscope#97

### PR DESCRIPTION
**Pull request by indianbhaji**
Narita International Airport (成田国際空港 Narita Kokusai Kūkō) (IATA: NRT, ICAO: RJAA), also known as Tokyo Narita Airport, formerly and originally known as New Tokyo International Airport (新東京国際空港 Shin Tōkyō Kokusai Kūkō), is an international airport serving the Greater Tokyo Area of Japan. It is located approximately 60 kilometres (37 mi) east of central Tokyo in Chiba Prefecture, straddling the border between the city of Narita and the adjacent town of Shibayama.

Narita is the predominant international airport in Japan, handling around 50% of the country's international passenger traffic and 60% of its international air cargo traffic. As of 2013, Narita was the second-busiest passenger airport in Japan (after Haneda Airport in Tokyo), and was the tenth-busiest air freight hub in the world. Its 4,000-metre (13,123 ft) main runway shares the record for longest runway in Japan with the second runway at Kansai International Airport in Osaka.

Narita serves as the main international hub of Japan Airlines, All Nippon Airways and Nippon Cargo Airlines, and as a hub for low-cost carriers Jetstar Japan, Peach and Vanilla Air. It also serves as an Asian hub for Delta Air Lines and United Airlines.

From [Wikipedia](https://en.wikipedia.org/wiki/Narita_International_Airport)

Also known as zlsa/atc#597

Resolves #103.

### CODE REVIEW

File: assets/airlines/jjp.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "callsign": {
+        "name": "Orange Liner",
+        "length": 3
+    },
+    "fleets": {
+        "default": [
+            ["A320", 20]
+        ]
+    }
+}
```
Comment (erikquinn) [inline assets/airlines/jjp.json]: be sure to end all files with a new line.

File: assets/airlines/sjo.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "callsign": {
+        "name": "Jey Spring",
+        "length": 3
+    },
+    "fleets": {
+        "default": [
+            ["B738", 3]
+        ]
+    }
+}
```
Comment (erikquinn) [inline assets/airlines/sjo.json]: be sure to end all files with a new line.

File: assets/airlines/vnl.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "callsign": {
+        "name": "Vanilla",
+        "length": 3
+    },
+    "fleets": {
+        "default": [
+            ["A320", 9]
+        ]
+    }
+}
```
Comment (erikquinn) [inline assets/airlines/vnl.json]: be sure to end all files with a new line.

File: assets/airports/airportLoadList.js [hunk 4]
```diff
@@ -288,6 +288,11 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             wip: false
         },
         {
+            icao: 'rjaa',
+            level: 'hard',
+            name: 'Tokyo Narita International Airport'
```
Comment (erikquinn) [inline assets/airports/airportLoadList.js]: please add `wip: true` here.

File: assets/airports/rjaa.json [hunk 5]
```diff
@@ -0,0 +1,1153 @@
+{
+    "radio": {
+        "twr": "Narita Tower",
+        "app": "Tokyo Approach",
+        "dep": "Tokyo Departure"
+    },
+    "icao": "RJAA",
+    "iata": "NRT",
+    "magnetic_north": 0,
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: `-7.3`, according to AIS Japan.

File: assets/airports/rjaa.json [hunk 6]
```diff
@@ -0,0 +1,1153 @@
...
+    "ctr_radius": 80,
+    "ctr_ceiling": 27000,
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: Is this the actual ceiling of the approach control's airspace? Or else, is there a good reason for it to be so high?

File: assets/airports/rjaa.json [hunk 7]
```diff
@@ -0,0 +1,1153 @@
...
+    "initial_alt": 3000,
+    "position": ["N35d45m55.00", "E140d23m08.00"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N35d45m55.00", "E140d23m08.00"],
+    "has_terrain": false,
+    "wind": {
+        "angle": 250,
+        "speed": 0
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: calm wind... I like it!

File: assets/airports/rjaa.json [hunk 8]
```diff
@@ -0,0 +1,1153 @@
...
+    },
+    "airspace": [
+        {
+            "floor": 			0,
+            "ceiling": 		    27000,
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: Please remove all hard tabs.

File: assets/airports/rjaa.json [hunk 9]
```diff
@@ -0,0 +1,1153 @@
...
+            "exitPoints": {
+                "SUNNS": [["SUNNS", "A190+"]]
+            },
+            "draw": [
+                ["NRE", "HKE", "ROUSY", "SUNNS*"]
+            ]
+        }
+    },
+
+    "stars": {
```
Comment (erikquinn) [inline assets/airports/rjaa.json]: I'm only seeing `ELGAR`, `TYLER`, `NORMA`, and `GEMIN` arrivals, and nothing on the AIS for `ABBOT`, `SWAMP`, and `LAKES`?

File: assets/airports/rjaa.json (hunk 6 above)
Comment (erikquinn) [inline assets/airports/rjaa.json]: Also note that this is still taken in hundreds of feet, so you would want it to take the form of `270`.

File: assets/airports/rjaa.json (hunk 9 above)
Comment (indianbhaji) [inline assets/airports/rjaa.json]: They are on there, and they're also on my charts from Navigraph

Comment (erikquinn) [inline assets/airports/rjaa.json]: Yup, my mistake!

**Pull request merged**